admin/templates/spam/email.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}
+    <title>Email</title>
+{% endblock title %}
+{% block content %}
+    <div class="container-fluid">
+        <div class="row">
+            <div class="btn-group col-sm-12">
+                <a href="javascript:window.history.go(-2);"
```

Comment:
It is possible. I've experimented with a few different options but I couldn't do it without using the url. Any good resources?